apps/examples/hello_tfmicro/Makefile

```diff
@@ -0,0 +1,143 @@
+###########################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
```

Comment:
Please check all of newly added files which have Samsung license.

Reply:
sorry, previously i missed this instance.